core/src/main/java/org/apache/calcite/rel/metadata/RelMdColumnOrigins.java

```diff
@@ -253,4 +264,24 @@ public Void visitInputRef(RexInputRef inputRef) {
     }
     return set;
   }
+
+  private Set<RelColumnOrigin> getMultipleColumns(RexNode rexNode, RelNode input,
+      final RelMetadataQuery mq) {
+    // Anything else is a derivation, possibly from multiple
+    // columns.
```

Comment:
This comment should be added in the caller of this method.

Reply:
> This comment should be added in the caller of this method.

Thanks, update it.